test002.c:6: エラー: `a' が再宣言されています.
test002.c:5: エラー: ここで宣言されていました.
